src/runtime_src/core/edge/drm/zocl/include/zocl_bo.h

```diff
@@ -3,7 +3,7 @@
  * A GEM style (optionally CMA backed) device manager for ZynQ based
  * OpenCL accelerators.
  *
- * Copyright (C) 2019 Xilinx, Inc. All rights reserved.
+ * Copyright (C) 2019-2020 Xilinx, Inc. All rights reserved.
```

Comment:
revert this.

Reply:
I moved this file to include/ directory. No need to update copyright in this case. Right?

Reply:
> 
> 
> I moved this file to include/ directory. No need to update copyright in this case. Right?

I think that you only update the copyright year when you have changed the content of the file.

Reply:
Done.